src/util/paths.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
    */
   normalizeFileDataUrl (fileDataUrl) {
     const isAbsolutePath = path.isAbsolute(fileDataUrl)
-    const isUrl = urlRegex().test(fileDataUrl)
+    const isUrl = urlRegex({strict:true}).test(fileDataUrl)
```

Comment:
you say `strict: false` in your comment above, but it's `true` here - is one of these a type?